tests/foreman/api/test_contentview.py

```diff
@@ -1730,59 +1451,59 @@ def test_negative_non_readonly_user_actions(self):
             entity for entity in cv_permissions_entities if entity.name in user_cv_permissions
         ]
         entities.Filter(
-            organization=[self.org], permission=user_cv_permissions_entities, role=role
+            organization=[module_org], permission=user_cv_permissions_entities, role=role,
         ).create()
         # create a user and assign the above created role
         entities.User(
-            organization=[self.org], role=[role], login=user_login, password=user_password
+            organization=[module_org], role=[role], login=user_login, password=user_password
         ).create()
-        # Create new content view entity using admin user
-        content_view = entities.ContentView(organization=self.org).create()
         cfg = get_nailgun_config()
         cfg.auth = (user_login, user_password)
         # Check that we cannot read our content view with custom user
-        with self.assertRaises(HTTPError):
+        with pytest.raises(HTTPError):
             entities.ContentView(cfg, id=content_view.id).read()
         # Check that we have permission to remove the entity
         entities.ContentView(cfg, id=content_view.id).delete()
-        with self.assertRaises(HTTPError):
+        with pytest.raises(HTTPError):
             entities.ContentView(id=content_view.id).read()
 
 
 @pytest.mark.skip_if_open("BZ:1625783")
-class OstreeContentViewTestCase(APITestCase):
+class TestOstreeContentView:
     """Tests for ostree contents in content views."""
 
-    @classmethod
     @skip_if_os('RHEL6')
-    def setUpClass(cls):  # noqa
+    @fixture(scope='class', autouse=True)
+    def setupclass(self, request, module_product):
```

Comment:
Consider giving this function a more descriptive name, now that the name isn't tied to unittest calling it.

Reply:
makes sense, kept that as an example so that others can understand while porting their components.

Reply:
Updated btw to make it more unlike unittest.